core/sql/optimizer/RelGrby.h

```diff
@@ -552,7 +560,9 @@ class GroupByAgg : public RelExpr
   // ---------------------------------------------------------------------
   ItemExpr    * groupExprTree_;
   ValueIdSet  groupExpr_;
-
+  
+  ItemExpr  * extraGrpOrderby_;
+  ValueIdSet  extraOrderExpr_;
```

Comment:
I am confused how an order expression can be stored in a ValueIdSet. Should this be a ValueIdList instead?

The order (A,B) is different from (B,A) but both would be stored in the same ValueIdSet.

This same question applies to change in OptPhysRelExpr.cpp. There we take a ValueIdSet, convert it to a List and make a order requirement out of it. If order is important maybe it should start as List. If order is not important then we could add an arrangement requirement instead of a order requirement.